src/main/java/IssuesPage.java

```diff
@@ -0,0 +1,85 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class IssuesPage extends BasePage {
+
+    public IssuesPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void newIssueBtn() throws InterruptedException {
+        WebElement newIssueBtn = driver.findElement(By.xpath("//a[@href='/HW10Test/TEST/issues/new/choose' and @class='btn btn-primary' and @role='button' and @data-hotkey='c']"));
+        newIssueBtn.click();
+        Thread.sleep(2500);
```

Comment:
please avoid explicitly waits - use implicitly waits if needed. It's not a good idea to add waits everywhere. You need use it when you face with nosuchelementexeption